pkg/vrpqstatus/status_test.go

```diff
@@ -188,4 +188,22 @@ var _ = Describe("status", func() {
 		Expect(vrpq.Status.Conditions[0].LastTransitionTime).ShouldNot(Equal(origTime))
 	})
 
+	It("should update the message status", func() {
```

Comment:
If we add a message for a failed query, e.g. "Query failed", we could also add a test here

Reply:
No need to update here; statust_test.go only tests whether the function updates the message or not. Regardless of the information contained in the message, it will always be updated. We can update in the query_reconciler_test.go